core/commands/ipns.go

```diff
@@ -94,7 +96,7 @@ Resolve the value of a dnslink:
 		var name string
 		if len(req.Arguments()) == 0 {
 			if n.Identity == "" {
-				res.SetError(errors.New("identity not loaded"), cmds.ErrNormal)
+				res.SetError(errors.New("Identity not loaded"), cmdkit.ErrNormal)
 				return
```

Comment:
Is there a reason for this change?  Errors should generally always start with a lowercase.